app/styles/toast.less

```diff
@@ -1,60 +1,114 @@
-@keyframes slide-up {
-  0% {
+@keyframes slide-in {
+  from {
     opacity: 0;
-    transform: translate(-50%, 10%);
+    transform: translate(0, -50%);
   }
 
-  100% {
+  to {
     opacity: 1;
-    transform: translate(-50%, 0);
+    transform: translate(0, 0);
   }
 }
 
-#toast-container.upf-toastr--container {
+.toasts-container {
+  position: fixed;
   display: flex;
-  justify-content: center;
   flex-direction: column;
-
-  position: fixed;
-  bottom: 0;
-  left: 50%;
-  transform: translateX(-50%);
-
-  width: 100%;
+  gap: @spacing-xx-sm;
+  top: 56px;
+  right: @spacing-x-sm;
   z-index: 1100;
 
   .toast {
-    margin-bottom: @spacing-xx-sm;
-    max-width: 700px;
-    position: relative;
-    margin-left: 50%;
-    margin-right: 50%;
-    transform: translateX(-50%);
-    align-items: center;
-  }
-
-  .toast--hidden {
     display: flex;
-    opacity: 0;
-    animation: none;
-  }
+    flex-direction: column;
+    width: 350px;
+    border: 1px solid var(--color-border-default);
+    border-radius: var(--border-radius-md);
+    overflow: hidden;
+    background-color: var(--color-gray-50);
+    box-shadow: var(--box-shadow-lg);
 
-  .toast--visible {
-    animation: slide-up 0.5s 1;
-    z-index: 20;
-    display: flex;
-  }
+    .counter {
+      height: 3px;
```

Comment:
`@padding-xx-sm / 2`
:trollface:

Reply:
For this one, it is `@padding-xxx-sm / 2` xD (missing one "x")